tests/unit/models/transactions/test_xchain_create_bridge.py

```diff
@@ -0,0 +1,166 @@
+            issuing_chain_door=_ACCOUNT,
+            issuing_chain_issue=IssuedCurrency(currency="USD", issuer=_ACCOUNT),
+        )
+        with self.assertRaises(XRPLModelException):
+            XChainCreateBridge(
+                account=_ACCOUNT,
+                fee=_FEE,
+                sequence=_SEQUENCE,
+                xchain_bridge=bridge,
+                signature_reward="200",
+            )
+
+    def test_iou_xrp_bridge(self):
+        bridge = XChainBridge(
+            locking_chain_door=_ACCOUNT,
+            locking_chain_issue=IssuedCurrency(currency="USD", issuer=_ISSUER),
+            issuing_chain_door=_ACCOUNT,
+            issuing_chain_issue=XRP(),
+        )
+        with self.assertRaises(XRPLModelException):
+            XChainCreateBridge(
+                account=_ACCOUNT,
+                fee=_FEE,
+                sequence=_SEQUENCE,
+                xchain_bridge=bridge,
+                signature_reward="200",
+            )
+
+    def test_account_not_in_bridge(self):
+        bridge = XChainBridge(
+            locking_chain_door=_ACCOUNT,
+            locking_chain_issue=XRP(),
+            issuing_chain_door=_ACCOUNT2,
+            issuing_chain_issue=XRP(),
+        )
+        with self.assertRaises(XRPLModelException):
+            XChainCreateBridge(
+                account=_GENESIS,
+                fee=_FEE,
+                sequence=_SEQUENCE,
+                xchain_bridge=bridge,
+                signature_reward="200",
+            )
+
+    def test_iou_iou_min_account_create_amount(self):
+        bridge = XChainBridge(
+            locking_chain_door=_ACCOUNT,
+            locking_chain_issue=IssuedCurrency(currency="USD", issuer=_ISSUER),
+            issuing_chain_door=_ACCOUNT2,
+            issuing_chain_issue=IssuedCurrency(currency="USD", issuer=_ACCOUNT2),
+        )
+        with self.assertRaises(XRPLModelException):
+            XChainCreateBridge(
+                account=_ACCOUNT,
+                fee=_FEE,
+                sequence=_SEQUENCE,
+                xchain_bridge=bridge,
+                signature_reward="200",
+                min_account_create_amount="1000000",
+            )
+
+    def test_invalid_signature_reward(self):
+        bridge = XChainBridge(
+            locking_chain_door=_ACCOUNT,
+            locking_chain_issue=XRP(),
+            issuing_chain_door=_GENESIS,
+            issuing_chain_issue=XRP(),
+        )
+        with self.assertRaises(XRPLModelException):
+            XChainCreateBridge(
+                account=_ACCOUNT,
+                fee=_FEE,
+                sequence=_SEQUENCE,
+                xchain_bridge=bridge,
+                signature_reward="hello",
+                min_account_create_amount="1000000",
+            )
+
+    def test_invalid_min_account_create_amount(self):
+        bridge = XChainBridge(
+            locking_chain_door=_ACCOUNT,
+            locking_chain_issue=XRP(),
+            issuing_chain_door=_GENESIS,
```

Comment:
I'm not really sure why it matters to use `_GENESIS` sometimes and `_ACCOUNT2` sometimes. Can we just use one account as the secondary one, or is there an important distinction?

Reply:
XRP bridges use `_GENESIS`, IOU bridges use another account (they can also use the genesis account).